### PR TITLE
Route

### PR DESCRIPTION
**Pull request by NtGZeko**

### CODE REVIEW

**Comment by NtGZeko** [comment]
fixed from github

**Review by FrancoisSalmon — DISMISSED** [review]
ok

File: backend/src/router.js [hunk 1]
```diff
@@ -10,7 +14,15 @@ router.put("/players/:id", PlayerController.edit);
 router.post("/players", PlayerController.add);
 router.delete("/players/:id", PlayerController.delete);
 
+router.get("/staffs", StaffController.browse);
+router.get("/staffs/:id", StaffController.browse);
```
Comment (LuigiMenez) [inline backend/src/router.js:18]: Useless

File: backend/src/router.js [hunk 2]
```diff
@@ -10,7 +14,15 @@ router.put("/players/:id", PlayerController.edit);
...
+router.get("/staffs/:id", StaffController.read);
+router.put("/staffs/:id", StaffController.edit);
+router.post("/staffs", StaffController.add);
+router.delete("/staffs/:id", StaffController.delete);
+
 router.get("/items", ItemController.browse);
+router.get("/items/:id", ItemController.browse);
```
Comment (LuigiMenez) [inline backend/src/router.js:25]: Useless

**Review by LuigiMenez — CHANGES_REQUESTED** [review]
Routes useless

**Review by FrancoisSalmon — APPROVED** [review]
ok

**Review by LoicBrassart — APPROVED** [review]
SnE

**Pull request merged**